src/api.ts

```diff
@@ -120,7 +126,7 @@ export interface StaticPlatformPlugin extends PlatformPlugin {
    *
    * @param {(foundAccessories: AccessoryPlugin[]) => void} callback
    */
-  accessories(callback: (foundAccessories: AccessoryPlugin[]) => void): void;
+  accessories(callback: (foundAccessories: Array<AccessoryPlugin & AccessoryConfig>) => void): void;
```

Comment:
I think this typing doesn't really represent reality (and might even break current implementations?).

The weird thing is, that we really only require the `name: string` property (and optionally check for the `uuid_base?: string` property). Using `AccessoryConfig` though will add the `accessory: AccessoryName | AccessoryIdentifier` requirement and the optional `_bridge?: BridgeConfiguration` property which both are never used.

  name: string;
   uuid_base?: string;

Reply:
Resolving this would make #2815 obsolete and resolve #2771